create-aliquot.avsc

```diff
@@ -0,0 +1,84 @@
+{
+  "type": "record",
+  "name": "CreateAliquot",
+  "namespace": "uk.ac.sanger.psd",
+  "fields": [
+     {
+      "name": "messageUuid",
+      "doc": "Unique message ID.",
+      "type": "uuid"
```

Comment:
This is a logical type rather than a top level type: https://avro.apache.org/docs/1.11.1/specification/#uuid

You need to specify it like the timestamp is below, so like:

```suggestion
      "type": {
        "type": "string",
        "logicalType": "uuid"
      }
```